.mailmap

```diff
@@ -1,26 +1,38 @@
+Alexander Hulpke <hulpke@math.colostate.edu> <hulpke@colostate.edu>
+Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alex-konovalov@users.noreply.github.com>
 Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alexk@cs.st-andrews.ac.uk>
 Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alexk@mcs.st-andrews.ac.uk>
-Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> <alex-konovalov@users.noreply.github.com>
 Andreas Distler <adistler@fc.ul.pt> <andreas@mcs.st-and.ac.uk>
 Burkhard Höfling <burkhard@hoefling.name> <bh11@users.noreply.github.com>
 Chris Jefferson <caj21@st-andrews.ac.uk>
 Chris Jefferson <caj21@st-andrews.ac.uk> <chris@bubblescope.net>
 Chris Wensley <cdwensley.maths@btinternet.com> <cwensley1234@btinternet.com>
 Christopher Russell <cr66@st-andrews.ac.uk>
+Dmitrii Pasechnik <dimpase@gmail.com> <dima@pasechnik.info>
 Dmitrii Pasechnik <dimpase@gmail.com> <Dmitrii.Pasechnik@cs.ox.ac.uk>
+Dominik Bernhardt <dominik.bernhardt@rwth-aachen.de> <dominik.bernhardt@rwth.aachen.de>
 Frank Lübeck <frank.luebeck@math.rwth-aachen.de> <Frank.Luebeck@math.rwth-aachen.de>
-Markus Pfeiffer <markus.pfeiffer@st-andrews.ac.uk> <markus.pfeiffer@morphism.de>
-Markus Pfeiffer <markus.pfeiffer@st-andrews.ac.uk> <markuspf@users.noreply.github.com>
-Markus Pfeiffer <markus.pfeiffer@st-andrews.ac.uk> <markusp@mcs.st-andrews.ac.uk>
-Max Horn <max@quendi.de> <max.horn@math.uni-giessen.de>
-Max Neunhöffer <neunhoef@mcs.st-and.ac.uk>
-James Mitchell <jdm3@st-and.ac.uk> <jdm3@st-andrews.ac.uk>
+Gábor Horváth <ghorvath@science.unideb.hu> <hungaborhorvath@gmail.com>
 James Mitchell <jdm3@st-and.ac.uk> <james-d-mitchell@users.noreply.github.com>
+James Mitchell <jdm3@st-and.ac.uk> <jdm3@st-andrews.ac.uk>
 James Mitchell <jdm3@st-and.ac.uk> <jenkyfunds@gmail.com>
 James Williams <flowersinabox@hotmail.com>
 John McDermott <jjm@mcs.st-andrews.ac.uk> <jjm@mcs.st-and.ac.uk>
+Laurent Bartholdi <laurent.bartholdi@gmail.com>
+Lucas Wollenhaupt <github@wucas.de> <git@wucas.de>
+Lucas Wollenhaupt <github@wucas.de> <reg-github@wucas.de>
```

Comment:
@wucas Please let me know if `Lucas Wollenhaupt <github@wucas.de>` is OK as your "preferred" git "identity" (or feel free to suggest something else)

Reply:
Yes that's alright, thanks!